user_guide_src/source/installation/installing_composer.rst

```diff
@@ -82,7 +82,25 @@ Whenever there is a new release, then from the command line in your project root
 
     composer update
 
-Read the :doc:`upgrade instructions <upgrading>`, and check Breaking Changes and Enhancements.
+Read the :doc:`upgrade instructions <upgrading>` and :doc:`change log <../changelogs/index>`,
+and check Breaking Changes and Enhancements.
+
+Upgrading to a Specified Version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For example, you may want to upgrade to v4.4.8 after v4.5.0 has been released.
```

Comment:
```suggestion
For example, you may want to upgrade from v4.4.7 to v4.4.8 after v4.5.0 has been released.
```
I think this would explain the case a bit better.